docs/vectorizer-worker.md

```diff
@@ -0,0 +1,234 @@
+
+# Run vectorizers in a self-hosted database using pgai vectorizer worker
```

Comment:
This file is a bit confusing now, it previously only covered the python vectorizer worker and now also contains database hosting? Is it supposed to be just a general self hosting guide? Then I think we should expand and structure it in two parts: 
1. The database
2. The vectorizer worker